Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -374,6 +374,14 @@ SeedFinder<external_spacepoint_t, platform_t>::getCompatibleDoublets(
         const float iDeltaR = std::sqrt(iDeltaR2);
         const float cotTheta = deltaZ * iDeltaR;
 
+        // discard bottom-middle dublets in a certain (r, eta) region according
+        // to detector specific cuts
+        if constexpr (candidateType == Acts::SpacePointCandidateType::eBottom) {
```

Comment:
This is used many times, it could potentially be centralized to 

```cpp
static constexpr bool isBottomCandidate = candidateType == Acts::SpacePointCandidateType::eBottom;
```

but not critical.